packages/features/bookings/lib/handleNewBooking/loadUsers.ts

```diff
@@ -85,3 +85,5 @@ export const findUsersByUsername = async ({
     };
   });
 };
+
+export type AwaitedLoadUsers = Awaited<ReturnType<typeof loadUsers>>;
```

Comment:
I think it's better to call it `LoadedUsers` or something similar.

Awaited doesn't make sense in name,